docs/language/operators/README.md

```diff
@@ -711,13 +711,12 @@ id.orig_h     id.orig_p id.resp_h       id.resp_p orig_bytes resp_bytes total_by
 
 #### Example #2:
 
-As noted above, shorthand syntax may be used, such as leaving out the optional
-`put` keyword or calling a function without specifying a field name in which
-to store the value it returns. Using both of these shortcuts, here we create
-a field to hold the lowercase representation of another field value.
+As noted above the `put` keyword is entirely optional. Here we omit
+the `put` keyword and create a new field to hold the lowercase
+representation of another field value.
 
 ```mdtest-command zed-sample-data/zeek-default
-zq -f table 'cut method | to_lower(method)' http.log.gz
+zq -f table 'cut method | to_lower:=to_lower(method)' http.log.gz
```

Comment:
```suggestion
zq -f table 'cut method | lower_method:=to_lower(method)' http.log.gz
```

It's a minor thing, but I figure we might as well use a field name different from the function name. If we thought the function names made for good field names we'd probably have had less consensus on proceeding with this PR. :smile: 

Note that it'll require adjusting the field header in the expected output. GitHub won't let me do that via this Suggestion feature since it's too far from the lines being changed. :roll\_eyes: